README.md

```diff
@@ -1,45 +1,17 @@
 # Using u-boot on Creator (Ci40) Marduk platform
 
-### How to build/cross-compile for Ci40:
+## Toolchain setup
+If you are using the toolchain produced by OpenWrt you must setup the environment as explained
+[here](https://wiki.openwrt.org/doc/devel/crosscompile).
 
-	$ export CROSS_COMPILE=/path/to/mips-toolchain/mips-toolchain-prefix
-	$ make pistachio_marduk_defconfig
-	$ make
+For the MIPS official toolchain see
+[here](https://community.imgtec.com/developers/mips/tools/codescape-mips-sdk/download-codescape-mips-sdk-essentials/).
 
-This will generate u-boot-pistachio-nor.img
-
-Note: Using OpenWrt's toolchain toolchain-mipsel_mips32_gcc-5.2.0_musl-1.1.11 will require [this patch](http://lists.denx.de/pipermail/u-boot/2015-July/217911.html) to be applied.
-
-### How to flash on Ci40:
-
-1. [Load OpenWrt](https://github.com/IMGCreator/openwrt/blob/master-pistachio/README.md)
-
-2. Erase and write u-boot image on bootloader partition of Ci40
-
-		$ flashcp -v u-boot-pistachio-nor.img /dev/mtd0
-
-	Note: flashcp needs to be manually selected in OpneWrt menuconfig
+## Configure and build it
+	$ make CROSS_COMPILE=<toolchain-prefix> pistachio_marduk_config
```

Comment:
Both will work but I guess defconfig is the newer way of doing it so I shall switch to that.